client/pages/index.tsx

```diff
@@ -39,9 +45,9 @@ const Home = () => {
     intervalRef.current = setInterval(
       () => {
         takePhoto();
-        setTimeout(donePhoto, 2 * 60 * 1000);
+        setTimeout(donePhoto, 2 * 1 * 1000);
```

Comment:
hey eri! this setInterval is for testing so we can emulate the photo being taken and then after a while we take another photo -- this will be removed later, so i think we can leave the magic numbers here for now?